main.cpp

```diff
@@ -51,107 +51,108 @@ arm.flex();                             // 3)
 arm.rotateCounterClockwise();           // 4) demonstrates camelCase
 
 
-//  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  1) 
+//  Noun: car
+//  action 1: the car turns on
+car.start();
+//  action 2: the car moves forward
+car.moveForward();
+//  action 3: the car stops
+car.stop();
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: chef
+//  action 1: the chef collects ingredients
+chef.collectIngredients();
+//  action 2: the chef dices vegetables
+chef.diceVeggies();
+//  action 3: the chef burns the onions
+chef.burnOnions();
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: teacher
+//  action 1: the teacher prepares the lesson
+teacher.prepareLesson();
+//  action 2: the teacher grades tests
+teacher.gradeTest();
+//  action 3: the teacher tutors after school
+teacher.tutor();
 //  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: pianist
+//  action 1: the pianist plays a C Maj chord   
+pianist.playCMaj();
+//  action 2: the pianist plays a glissando
+pianist.gliss();
+//  action 3: the pianist activates the sustain pedal
+pianist.activateSustain();
 //  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: ghost
+//  action 1: the ghost opens the closet door
+ghost.openCloset();
+//  action 2: the ghost turns off a light
+ghost.turnOffLight();
+//  action 3: the ghost breaks a vase
+ghost.breakVase();
 //  6)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: cat
+//  action 1: the cat sleeps
+cat.sleep();
+//  action 2: the cat eats
+cat.eat();
+//  action 3: the cat walks on the piano
+cat.walkOnPiano();
 
 //  7)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: quarterback
+//  action 1: the quarterback throws the ball
+quarterback.throwBall();
+//  action 2: the quarterback runs the ball
+quatrerback.run();
+//  action 3: the quarterback scores
+quarterback.score();
 //  8)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: baker
+//  action 1: the baker kneads the dough
+baker.knead();
+//  action 2: the baker bakes a cake
+baker.bakeCake();
+//  action 3: the baker decorates the cake
+baker.decorateCake();

```

Comment:
As explained in the instructions:
Here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.

Do not create an 11th noun.

Once you have completed the above-requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message.